src/V3Global.cpp

```diff
@@ -87,13 +87,14 @@ void V3Global::readFiles() {
     }
 }
 
-string V3Global::debugFilename(const string& nameComment, int newNumber) {
+string V3Global::debugFilename(const string& nameComment, int newNumber) VL_MT_SAFE {
```

Comment:
VL_MT_SAFE_EXCLUDES.  Likewise any function you mark as safe here that acquires a lock that is in the shared structure (versus static) should be an EXCLUDES.